src/status_im/contexts/communities/actions/community_options/view.cljs

```diff
@@ -139,6 +139,7 @@
   [id token-gated? intro-message]
   [[(when-not token-gated? (view-members id))
     (when-not token-gated? (view-rules id intro-message))
+    (mark-as-read id)
```

Comment:
Add mark as read to specated communities, they are counted in the read messages and it's annoying to have to read all channels